### PR TITLE
monitoring v2 tweaks

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
#1689 - set Grafana `persistence.enabled` to `true` when pvc or statefulset storage is configured

#1667 - add Advanced nav group to monitoring
Screen Shot 2020-10-22 at 4 36 03 PM

#1668 - add tooltips to `aggregateToDefaultRoles` and `createDefaultRoles` checkboxes
Screen Shot 2020-10-23 at 11 33 22 AM
Along with adding tooltips, I tweaked some of the `text-label` class usage to be more consistent with other inputs:
Screen Shot 2020-10-23 at 11 44 24 AM
Screen Shot 2020-10-23 at 11 44 33 AM

#1660 - enable `k3sServer` when `k3sControllerManager`, `k3sScheduler`, `k3sProxy` are enabled
The basic configuration aspect of this ticket is completed; the dashboard will enable k3sServer when it enables `k3sControllerManager`, `k3sScheduler`, and `k3sProxy` 
The advanced configuration portion of this ticket is dependent on #909

### CODE REVIEW

**Review by westlywright — APPROVED** [review]

**Review by lvuch — COMMENTED** [review]
Review submitted via commit

File: assets/styles/base/_typography.scss [hunk 1]
```diff
@@ -37,10 +37,43 @@ H6 {
...
   font-style: normal;
   letter-spacing: 0em;
   margin: 0;
 }
+
+UL {
+  LI {
+    margin: 10px 0;
+  }
+}
```
Comment (vincent99) [inline assets/styles/base/_typography.scss:52]: You really want this on every LI everywhere by default?  i.e. the left nav spacing seems to be getting out of hand..

image

File: assets/styles/base/_typography.scss [hunk 2]
```diff
@@ -37,10 +37,43 @@ H6 {
 }
 
 P {
-  font-size: 1.4em;
+  // font-size: 1.4em;
   line-height: 1.75em;
```
Comment (vincent99) [inline assets/styles/base/_typography.scss:41]: I don't know why `<p>` had a huge font to begin with, but shouldn't the line-height be removed too?

File: assets/translations/en-us.yaml [hunk 3]
```diff
@@ -711,7 +711,9 @@ monitoring:
     many: ReadWriteMany
     once: ReadWriteOnce
     readOnlyMany: ReadOnlyMany
-  aggregateDefaultRoles: Aggregate to Default Kubernetes Roles
+  aggregateDefaultRoles: 
+    label: Aggregate to Default Kubernetes Roles
+    tip: Adds labels to the ClusterRoles deployed by the Monitoring chart to <a target="_blank" rel="noopener nofollow" href="https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles"> aggregate to the corresponding default k8s admin, edit, and view ClusterRoles.</a>
```
Comment (vincent99) [inline assets/translations/en-us.yaml:716]: `noreferrer`.  Always all 3 for external links.

**Review by vincent99 — CHANGES_REQUESTED** [review]
@lvuch see spacing comments, @mantis-toboggan-md just the rel=...noreferrer

**Pull request merged**